modules/persistent-storage-csi-efs-metrics-procedure.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-procedure_{context}"]
+= Enabling usage metrics
+
+To enable AWS EFS storage CSI usage metrics:
```

Comment:
Can we add the oc edit cli equivalent please?

Reply:
Done.